locales/en-US/notes.properties

```diff
@@ -1,5 +1,34 @@
-welcomeTitle2=Hello!
-welcomeText2=Welcome to this one-page notepad built in to Firefox. Browse the web. Jot down notes. It’s that simple.
+welcomeTitle3=Welcome to Firefox Notes!
+welcomeSubtitle3=A simple tool for taking notes while you browse.
+welcomeOpenNotes3=To open Notes:
+# LOCALIZATION NOTE (welcomeWindowsLinuxShortcut3): shortcut is the
+# key combination to open the sidebar
+welcomeWindowsLinuxShortcut3=On Windows, Linux: {shortcut}
+# LOCALIZATION NOTE (welcomeMacShortcut3): shortcut is the
+# key combination to open the sidebar
+welcomeMacShortcut3=On Mac: {shortcut}
+welcomeAccessNotes3=To access your notes from any laptop or desktop computer:
+# LOCALIZATION NOTE (welcomeSyncInfo3): sync is the label on the sync button
+welcomeSyncInfo3=Select {sync} to sign into your Firefox Account.
+# LOCALIZATION NOTE (welcomeFormatText3): `To format your text, use the toolbar or these commands.`
+# This part is in Bold
+welcomeFormatText3=To format your text,
+# LOCALIZATION NOTE (welcomeFormatUse3): `To format your text, use the toolbar or these commands`
+# This part is in normal font weight
+welcomeFormatUse3=use the toolbar or these commands:
+welcomeHeading3=Headings: Begin a line with `#` or `##` or `###` followed by a space to create a heading 1, heading 2, or heading 3
```

Comment:
Add a comment explaining to keep the backticks, and why they're used in this string